templates/admin/save_results.html.twig

```diff
@@ -0,0 +1,87 @@
+{% extends '@EasyAdmin/default/layout.html.twig' %}
+{% block content %}
+    <link rel="stylesheet" href="{{ asset('build/tournament.css') }}">
```

Comment:
The same: should work without this line, css should be included in tournament.js,